.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    allow:
+      - dependency-type: production
+    schedule:
+      interval: monthly
+    rebase-strategy: auto
```

Comment:
this was previously `disabled`, preventing automatic rebasing when the default branch changed. since this repo should be lower volume than monorail, its probably fine to enable.